src/corecounts.rs

```diff
@@ -97,12 +96,27 @@ mod tests {
 
     #[test]
     fn get_cuda_cores_custom() {
-        env::set_var(
+        temp_env::with_var(
```

Comment:
Is the temp_env addition required?

Reply:
Yes, else you couldn't have two tests testing the env variables.